server.js

```diff
@@ -0,0 +1,44 @@
+var express = require('express');
+var fs = require('fs');
+var bodyParser = require('body-parser');
+
+var app = express();
+var stringifyFile;
```

Comment:
we shouldn't use global variables when something is changed in endpoints - try to think how to skip global variables